test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -1574,13 +1574,29 @@ public void indexRandom(boolean forceRefresh, boolean dummyDocuments, boolean ma
         }
     }
 
+    /*
+     * This method deletes the given bogus documents
+     * @param bogusIds doc ids to be deleted
+     */
+    protected void deleteBogusDocs(Set<List<String>> bogusIds) {
```

Comment:
this function needs to have some validation checks like bogus docs exists or not, will update this